testcases.py

```diff
@@ -391,6 +391,8 @@ def check(self) -> TestResult:
             if hasattr(p, "tls_handshake_type"):
                 if p.tls_handshake_type == "1":
                     num_ch += 1
+            elif hasattr(p, "retransmission") or hasattr(p, "overlap"):
```

Comment:
Can you add a comment with a link to the issue here?

Reply:
Sure.  See 6d71932